docs/metrics.md

```diff
@@ -46,6 +53,17 @@ at the current time, they should not be used to report historical data such as:
 Metrics are also ephemeral, meaning there is no guarantee that they will be persisted for any amount of time. If you need
 a way to view and analyze historical data, consider the [workflow archive](workflow-archive.md) or reporting to logs.
 
+### Default Controller Metrics
+
+There are several controller-level metrics. These include:
+
+* `workflows_processed_count`: a count of all Workflow updates processed by the controller
```

Comment:
Please don't capitalize "Workflow" - it's should really be lowercase, same for workflow templates and cron workflows. It makes it harder to read.